demo.js

```diff
@@ -46,3 +64,70 @@ fluid.defaults("flock.demo.oscilOscil.synth", {
         }
     }
 });
+
+
+fluid.defaults("flock.demo.oscilOscil.nexusui", {
+    gradeNames: "fluid.viewComponent",
+
+    members: {
+        scope: null,
+        spectrogram: null
+    },
+
+    model: {
+        width: 300,
+        height: 100
+    },
+
+    selectors: {
+        scope: "#scope",
+        spectogram: "#spectogram"
+    },
+
+    events: {
+        onConnect: null,
+        onDisconnect: null
+    },
+
+    listeners: {
+        "onCreate.createElements": {
+            funcName: "flock.demo.oscilOscil.nexusui.createElements",
+            args: ["{that}", "{enviro}"],
+        },
+        "onConnect.connectScope": {
+            "this": "{that}.scope",
+            method: "connect",
+            args: "{enviro}.audioSystem.nativeNodeManager.outputNode"
+        },
+
+        "onConnect.connectSpectrum": {
+            "this": "{that}.spectogram",
+            method: "connect",
+            args: "{enviro}.audioSystem.nativeNodeManager.outputNode"
+        },
+
+        "onDisconnect.disconnectScope": {
+            "this": "{that}.scope",
+            method: "disconnect",
+            args: "{enviro}.audioSystem.nativeNodeManager.outputNode"
+        },
+
+        "onDisconnect.disconnectSpectrum": {
+            "this": "{that}.spectogram",
+            method: "disconnect",
+            args: "{enviro}.audioSystem.nativeNodeManager.outputNode"
+        }
+    }
+});
+
+flock.demo.oscilOscil.nexusui.createElements = function( that, enviro){
+
+    that.model.width = that.container.innerWidth();
```

Comment:
Models should always be changed via the [ChangeApplier](https://docs.fluidproject.org/infusion/development/changeapplierapi) so that notifications will fire to any components that are relaying or listening to changes of model values. So this can be changed to:

```that.applier.change("width", that.container.innerWidth());```